debuginfo/src/dwarf.rs

```diff
@@ -524,7 +524,11 @@ impl<'d, 'a> DwarfUnit<'d, 'a> {
             lines.reserve(rows.len());
 
             for row in rows {
-                let file = self.resolve_file(row.file_index)?.unwrap();
+                let file = self
+                    .resolve_file(row.file_index)
+                    .ok()
+                    .flatten()
```

Comment:
Since both usages now only require an option, can you change the function signature to `Option<FileInfo>`?